web/src/site-admin/SiteAdminExternalServicePage.tsx

```diff
@@ -167,6 +182,13 @@ export class SiteAdminExternalServicePage extends React.Component<Props, State>
                 {this.state.updatedOrError === true && (
                     <p className="alert alert-success user-settings-profile-page__alert">Updated!</p>
                 )}
+                {externalService?.webhookURL && (
+                    <div className="alert alert-info">
+                        <h3>Webhooks</h3>
+                        <p>Point webhooks for this external service at the following URL:</p>
```

Comment:
We don't call them "external services" in the product anymore:

<img width="557" alt="CleanShot 2020-05-13 at 16 34 44@2x" src="https://user-images.githubusercontent.com/1185253/81826127-bde71780-9537-11ea-8a95-1105a8cbaf50.png">

So:

```suggestion
                        <p>Point webhooks for this code host connection at the following URL:</p>
```